pkg/report/report.go

```diff
@@ -910,28 +910,33 @@ func (r *pdfReporter) createManagementSummary(parsedModel *types.Model, tempFold
 			{Value: float64(countLow), //Label: strconv.Itoa(countLow) + " Low",
 				Style: chart.Style{
 					FillColor: makeColor(rgbHexColorLowRisk()).WithAlpha(98),
-					//FontColor: makeColor(rgbHexColorLowRisk()),
-					FontSize: 65}},
+					FontStyle: chart.FontStyle{
```

Comment:
Font configuration was moved to a `FontStyle` struct.  This makes it easier to copy all font properties among styles.